tests/sqlparser_databricks.rs

```diff
@@ -277,3 +277,48 @@ fn parse_use() {
         );
     }
 }
+
+#[test]
+fn parse_databricks_struct_function() {
+    assert_eq!(
+        databricks()
```

Comment:
```suggestion
        databricks_and_generic()
```
thinking since generic accepts the same syntax? (same for the others)

Reply:
Yes, these are already tested with BigQuery, but I think it wouldn't hurt to add it here also.